app/Http/Controllers/RandomizeController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Mismatch;
+use App\Services\StatsdAPIClient;
+
+class RandomizeController extends Controller
+{
+    use Traits\ReviewMismatch, Traits\StatsTracker;
+
+    /**
+     * Instantiate a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct(StatsdAPIClient $statsd)
+    {
+        $this->middleware('simulateError');
+        $this->statsd = $statsd;
+    }
+
+    public function index()
+    {
+        $itemIds = Mismatch::with('importMeta.user')
+            ->where('review_status', 'pending')
+            ->whereHas('importMeta', function ($import) {
+                $import->where('expires', '>=', now());
+            })
+            ->inRandomOrder()
+            ->limit(15)
+            ->pluck('item_id')->toArray();
```

Comment:
Is this guaranteed to return 15 different items, or could the same item also be selected several times?

Reply:
I did a manual test and it returns always different items

Reply:
> I did a manual test and it returns always different items. I would need to test when there are less than 15 items to see what happens

For less than 15 items, it would select the same items with different order imo.

Reply:
For less than 15 items it should display the available ones, it's outlined in the ticket.

Reply:
I tried it out with a CSV file based on the example mismatches, and got a URL like this:

http://localhost:1337/results?ids=Q184746%7CQ9391%7CQ9391%7CQ184746%7CQ9391%7CQ184746%7CQ184746%7CQ184746%7CQ184746%7CQ9391%7CQ184746%7CQ184746%7CQ9391%7CQ9391%7CQ184746

The same item IDs are being repeated over and over again, because the query selects the item ID of 15 mismatches, not 15 different item IDs.